samples/python/kalman.py

```diff
@@ -81,12 +81,12 @@ def draw_cross(center, color, d):
 
             kalman.correct(measurement)
 
-            process_noise = kalman.processNoiseCov * np.random.randn(2, 1)
+            process_noise = sqrt(kalman.processNoiseCov[0,0]) * np.random.randn(2, 1)
             state = np.dot(kalman.transitionMatrix, state) + process_noise
 
             cv2.imshow("Kalman", img)
 
-            code = cv2.waitKey(100) % 0x100
+            code = cv2.waitKey(100)
```

Comment:
This change is fine for comparison with "-1", but masking with `0xff` is required for lines below (to work on all platforms):
```
if code in [27, ord('q'), ord('Q')]:
```